support/cas-server-support-pac4j/src/main/java/org/apereo/cas/support/pac4j/config/support/authentication/Pac4jAuthenticationEventExecutionPlanConfiguration.java

```diff
@@ -125,6 +125,7 @@ private void configureFoursquareClient(final Collection<BaseClient> properties)
     private void configureGoogleClient(final Collection<BaseClient> properties) {
         final Pac4jProperties.Google google = casProperties.getAuthn().getPac4j().getGoogle();
         final Google2Client client = new Google2Client(google.getId(), google.getSecret());
+        client.setName("Google");
```

Comment:
Good point.  Hardcoding the name presents the side effect that it makes it harder to change it later, in case someone wants "Google Login", "Google Authentication" or anything else. Preferences run wild around here :) 

One compromise would be that we should still externalize the name, yet if anyone wanted to change it to something else that would also mean some UI changes would be in order to make sure the switch matches. UI changes in this area almost always a requirement for deployments so this should not be too troublesome. 

Big picture, we might want to rethink the logic that it takes to calculate a type better so as not to not tie it directly with the name but for the time being, this should do OK.